content/bn/docs/concepts/security/_index.md

```diff
@@ -79,7 +79,8 @@ IaaS প্রদায়ক        | লিঙ্ক |
 আইবিএম ক্লাউড | https://www.ibm.com/cloud/security |
 মাইক্রোসফট আজওর | https://docs.microsoft.com/en-us/azure/security/azure-security |
 অরাকেল ক্লাউড ইন্ফ্রাস্ট্রাকচার | https://www.oracle.com/security |
-VMware vSphere | https://www.vmware.com/security/hardening-guides |
+টেনসেন্ট ক্লাউড | https://www.tencentcloud.com/solutions/data-security-and-information-protection |
+ভিএমওয়্যার ভিস্ফিয়ার | https://www.vmware.com/security/hardening-guides |
```

Comment:
Can someone please raise a PR to fix this url, its saying page not found 
<img width="1257" alt="image" src="https://github.com/user-attachments/assets/e36e3a77-066d-4e73-8a61-b0d81200b1fc" />

this is new url - https://www.vmware.com/solutions/security/hardening-guides